api/woeip/apps/air_quality/models.py

```diff
@@ -42,7 +42,7 @@ class Sensor(models.Model):
         ("PM10", "PM10"),
         ("PM2.5", "PM2.5"),
         ("μg/m3", "μg/m3"),
-        ("latlon", "latlon"),
+        ("lonlat", "lonlat"),
```

Comment:
- Appears to be breaking the migrations and will need to remain labeled as 'latlon'